dev-support/pysubmarine/gen-sdk.sh

```diff
@@ -29,6 +29,7 @@ submarine_dist_exists=$(find -L "${SUBMARINE_PROJECT_PATH}/submarine-dist/target
 if [[ -z "${submarine_dist_exists}" ]]; then
   cd "${SUBMARINE_PROJECT_PATH}"
   mvn clean package -DskipTests
+  cd "$FWDIR" # go back to FWDIR after packaging
```

Comment:
If lacking this command, there will be an error. To be more specific, in line 52, we cannot find swagger_config.json because we are in root dir of submarine but swagger_config.json is in ./dev-support